src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -619,6 +625,13 @@ public SDKClusterService getSdkClusterService() {
         return sdkClusterService;
     }
 
+    /**
+     * Updates the SDKClusterService. Called from {@link ExtensionsInitRequestHandler}.
+     */
```

Comment:
This makes sense. Not for this PR, we should also have a way to update the cluster info when an extension asks for it. (a.k.a lazy load).

Reply:
That is currently implemented with `state()` which actually does a cluster state request (although that can be filtered, see #354). We can add any other required calls as well when we need them.